index.html

```diff
@@ -0,0 +1,270 @@
+        <i class="fas fa-bars"></i>
+      </a>
+      <img src="img/logo.jpg" alt="youtube img">
+    </div>
+    <div class="col-md-6  search-bar">
+      <input type="text" name="text" placeholder="Search">
+      <button for="text"><i class="fas fa-search"></i></button>
+    </div>
+    <div class="col-md-3  news">
+      <a href="">
+        <i class="fas fa-video"></i>
+      </a>
+      <a href="">
+        <i class="fas fa-th"></i>
+      </a>
+      <a href="">
+        <i class="fas fa-bell"></i>
+      </a>
+      <a href="">
+        <i class="fas fa-user-circle"></i>
+      </a>
+    </div>
+  </header>
+  <div>
+    <div class="container">
+      <section class="main">
+        <div class="video-principal">
+          <iframe width="727" height="409" src="https://www.youtube.com/embed/g_0QtAwkVh8" frameborder="0"
+            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </div>
+        <div class="video-info">
+          <div id="info">
+            <h1>Is The New 2020 Ducati Superleggera V4 Worth €100,000?</h1>
+          </div>
+          <div id="info-1">
+            <div id="info-text">
+              <p class="info-p">480,905 views . 5 March 2020 </p>
+            </div>
+            <div id="top-buttons">
+              <a href="">
+                <i class="fas fa-thumbs-up"></i>
+              </a>
+              <p class="info-p">5560</p>
+              <a href="">
+                <i class="fas fa-thumbs-down"></i>
+              </a>
+              <p class="info-p">197</p>
+              <a href="">
+                <i class="fas fa-share"></i>
+              </a>
+              <p class="info-p">SHARE</p>
+              <a href="">
+                <i class="fas fa-square"></i>
+              </a>
+              <p class="info-p">SAVE</p>
+              <a href="">
+                <i class="fas fa-ellipsis-h"></i>
+              </a>
+            </div>
+          </div>
+        </div>
+        <hr>
+        <div id="chanel-info">
+          <div id="top-container-chanel-info">
+            <div id="video-owner">
+              <div>
+                <img id="img-owner" src="img/owner-img.jpg" alt="MOTOBOB Chanel">
+              </div>
+              <div id="inf-owner">
+                <h2>MOTOBOB</h2>
+                <p class="info-p">84.6K subscribers</p>
+              </div>
+            </div>
+            <div id="subscribe-button">
+              <div class="button">SUBSCRIBE</div>
+            </div>
+          </div>
+          <div id="chanel-description">
+            <h4>Sign up for my email newsletter of the best motorcycling stories here:</h4>
+            <a href="https://mailchi.mp/306039bd7519/motobob">
+              <span>https://mailchi.mp/306039bd7519/motobob</span>
+            </a>
+            <p class="info-p"><b><br><b><br>SHOW MORE</p>
+          </div>
+        </div>
+        <hr>
+        <b><br><b><br>
```

Comment:
This is not a correct use of `<b>` tag, we use `<b></b>` to make a text bold and it should have a closing tag. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b. Please select only the text that you want to highlight and not the whole sections of HTML code.